### PR TITLE
feat(ee-command): add support for skiff user bonus

### PR DESCRIPTION
**Pull request by McPizza0**
this PR adds support to give orgs bonuses
initially just domains

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#213** <a href="https://app.graphite.dev/github/pr/un/inbox/213?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: ee/apps/command/server/middleware/01.cors.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  options: H3CorsOptions
+): EventHandler<EventHandlerRequest, TResponse> => {
+  return defineEventHandler((event) => {
+    handleCors(event, options);
+    return handler(event);
+  });
+};
+
+export default corsEventHandler(() => {}, {
+  origin: process.env.WEBAPP_URL ? [process.env.WEBAPP_URL] : '*',
```
Comment (BlankParticle) [inline ee/apps/command/server/middleware/01.cors.ts:24]: this also needs to include the command app's url

File: ee/apps/command/server/trpc/trpc.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+const isAccountAuthenticated = trpcContext.middleware(({ next, ctx }) => {
+  if (!ctx.account || !ctx.account.id || !ctx.account.username) {
+    throw new TRPCError({
+      code: 'UNAUTHORIZED',
+      message: 'You are not logged in'
+    });
+  }
+  return next({
+    ctx: { ...ctx, account: ctx.account }
```
Comment (BlankParticle) [inline ee/apps/command/server/trpc/trpc.ts:18]: why are we doing this?
this is not necessary, as far as I can see

Comment (McPizza0) [reply ee/apps/command/server/trpc/trpc.ts:18]: additional layer of checks

Comment (BlankParticle) [reply ee/apps/command/server/trpc/trpc.ts:18]: I was talking about the `{ctx:{...ctx}}`, We don't need it

Comment (McPizza0) [reply ee/apps/command/server/trpc/trpc.ts:18]: this is how we do it in platform and other places
i think its as per trpc docs
but yes, does make more sense

**Review by BlankParticle — APPROVED** [review]

**Comment by McPizza0** [comment]
### Merge activity

* **Apr 3, 11:09 AM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/213).

**Pull request merged**